auth/api/http/policies/transport.go

```diff
@@ -33,6 +33,13 @@ func MakeHandler(svc auth.Service, mux *bone.Mux, tracer opentracing.Tracer) *bo
 		opts...,
 	))
 
+	mux.Put("/policies", kithttp.NewServer(
```

Comment:
Why PUT for deletion, and not DELETE?

Reply:
Since OpenAPI spec 3.0 doesn't accept request body for DELETE method type, I picked PUT over DELETE.

Reply:
What about additng a query parameter instead of the body? Would it make sense? Because it will definitely be more instructive in the code.

Reply:
What about additng a query parameter instead of the body? Would it make sense? Because it will definitely be more instructive in the code.

Reply:
(share operation needs to be done also with query param, to stay consistent)

Reply:
Since there is a list in the payload, we only can change this endpoint to remove only a single policy. I don't find removing policies one-by-one practical for the end-user, so I propose to keep it like this.